LIPS/lip-28.md

```diff
@@ -0,0 +1,106 @@
+3. **Tiebreaker Committee** is designed to overcome potential tie between the protocol and governance in the particular edge-case [[SPEC](https://github.com/lidofinance/dual-governance/blob/develop/docs/mechanism.md#tiebreaker-committee)]. In order for the Tiebreaker to be activated, the tie should be detected onchain first: 1) Dual Governance is in RageQuit state and the Lido on Ethereum protocol withdrawals are paused for a `TiebreakerActivationTimeout` duration; OR 2) the last time DG had been in Normal or Veto Cooldown state more than `TiebreakerActivationTimeout` time ago.
+    
+  Once activated, Tiebreaker can do two things:
+    
+ 1. Execute the blocked proposal in Dual Governance pipeline
+ 2. Unpause any of the paused protocol contracts
+  
+  Tiebreaker is proposed to be set up as “multisig of multisigs”, with 2/3 quorum on top multisig, and independent quorum settings for underlying multisigs.
+
+## Known risks and limitations
+
+The list of the known risks and limitations of the overall design and implementation details can be found here: https://github.com/lidofinance/dual-governance/blob/3e0f1ae5740ef8410e928f6cc106e3a5f45a5a75/docs/known-risks-and-limitations.md
+
+## Notes on deployment
+
+### Roles to be shifted
+
+In order for Dual Governance to be useful, it must have specific controls in the Lido on Ethereum protocol. There’s a number of “control types” used across Lido on Ethereum smart contracts (contract “ownership”, Aragon ACL roles, non-Aragon ACL roles are the main groups). The actual controls and control-over-controls is distributed between a small number of contracts: Aragon Voting, Aragon Agent, EVMScriptExecutor (EasyTrack motions execution gadget) and specific committees (i.e. GateSeal).
+
+The general idea of the transition to Dual Governance is: 1) Aragon Voting to renounce the in-protocol roles / pass role management to Aragon Agent, so that Aragon Voting can’t “do things” directly in the Lido on Ethereum protocol; 2) Aragon Agent to grant the roles for executing on its’ behalf to Dual Governance Executor contract; 3) EasyTrack setup remains as-is (Aragon Voting can change any settings, but can’t grant protocol-specific roles, which are managed by Aragon Agent under Dual Governance); 4) committees setup remains as-is.
+
+### Smoke test plan
+
+There’s a proposal to host a smoke test on Dual Governance smart contracts before proposing the DAO to switch the mechanics on in mainnet. Smoke test is to trial emergency activation and execution flows before the DAO moves the smart contract roles and permissions under the Dual Governance, allowing for contingencies where needed.
+
+1. Deploy the whole setup with temporary `emergency governance` multisig, so that the contracts can be properly set up before the DAO vote;
```

Comment:
better to say explicitly what entity is substituted with the `emergency governance` multisig

Reply:
there's no existing readymade thing, planned to have temp setup developer multisig; don't want to introduce yet another name to LIP + will be sharing more detailed plan on forum